### DISCUSSION TITLE
Import

### DISCUSSION THREAD

**Discussion by GerhardMes** [directus/directus#10546] [Feature Requests]
I searched how I can import e.g. a list of users from Excel or CSV to Directus. But could not find anything. In this case, I would need to upgrade the "standard" user collection. But such a feature would be interesting for content collection too. Especially when thinking to migrate e.g. my whole blog with several 100s' of blogposts - in two languages.

**Comment by rijkvanzanten** [comment]
Heya! You could either use regular POST requests to insert JSON through the regular API endpoints (https://docs.directus.io/reference/items/#create-multiple-items), or rely on the CSV/JSON import endpoint to inject data from a CSV file: https://docs.directus.io/reference/system/utilities/#import-data-from-file

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
